tests/ui_json_test.py

```diff
@@ -540,7 +540,7 @@ def test_input_file(tmp_path: Path):
     ):
         InputFile.read_ui_json("somefile.json")
 
-    with pytest.raises(TypeError, match="expected str, bytes or os.PathLike object"):
+    with pytest.raises(TypeError):
```

Comment:
**question:** why removing the test of the error message?

`match` can take a regex if it helps to have the test more resistant to minor changes in the message text

Reply:
Looks like the error message returned by pathlib in version 3.12 has changed substantially.

Reply:
ho, if the message is not coming from the code of the repo itself, checking for the error text is less relevant. As long as it captures a TypeError